faker/providers/barcode/ja_JP/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .. import Provider as BarCodeProvider
+
+
+class Provider(BarCodeProvider):
+    # Source of GS1 country codes: https://gs1.org/standards/id-keys/company-prefix
+    local_prefixes = (4, 5), (4, 9)
+
+    def jan(self, length=13):
```

Comment:
Can you please write docstrings with samples for the methods specific to the `ja_JP` provider? They are essentially just aliases and call `localized_ean*` under the hood, so something noting that fact will suffice.

Reply:
Fixed this on the last commit! (Did I get it right?)

Reply:
Please add summary lines to the docstrings. As for samples, only `jan()` requires an explicit sample section since it is the only one that accepts arguments. The methods `jan8()` and `jan13()` do not require one, and a sample section will automatically be generated for them. Just remove the `:return:` stuff in those two methods. You can read more about samples [here](https://faker.readthedocs.io/en/master/writing-docs.html).

Anyway, docstring should roughly follow this form.

```
def method(arg1, arg2):
    """Describe what method does in imperative mood.

   Notes and other info about method should be placed here.

    :sample: arg1='val1', arg2='val2'
    :sample: arg1=2, arg2=12
    """
   pass
```

Reply:
I got to generate the actual sphinx doc and looked around.
I fixed some references and added summery lines and sample lines, and I just went and added `:sample: ` to jan8() and jan13() too  (I thought it looks nicer on the doc).

Thank you for guiding me out!